source/Applications/PointCloudTutorial.md

```diff
@@ -0,0 +1,131 @@
+
+#### Copy selected data from GPU to system memory (Zivid-allocated)
+
+If you are only concerned about e.g. RGB color data of the point cloud, you can copy only that data to the system memory.
+```cpp
+auto rgba = frame.pointCloud().copyColorsRGBA(); /* Colors are copied from the GPU and into a
+Zivid::Array2D<Zivid::ColorRGBA>, which takes ownership of the data. */
+
+auto *dataPtr = const_cast<void *>(static_cast<const void *>(rgba.data())); /* Cast the data pointer as
+a void*, since this is what the OpenCV matrix constructor requires. */
+
+cv::Mat rgba(rgba.height(), rgba.width(), CV_8UC4, dataPtr); /* We wrap this block of data in an OpenCV
+matrix. This is possible since the layout of Zivid::ColorRGBA exactly matches the layout of CV_8UC4.
+No copying occurs in this step. */
+```
+
+####  Copy selected data from GPU to system memory (user-allocated)
+
+In the above example, ownership of the data was held by the returned ```Zivid::Array2D<>``` objects. Alternatively, you may provide a pre-allocated memory buffer to ```Zivid::PointCloud::copyData(dataPtr)```; the type of ```dataPtr``` defines what shall be copied (```PointXYZ```, ```ColorRGBA```, etc.).
+
+Now let us look at the exact same use case as above. However, this time, we allow OpenCV to allocate the necessary storage. Then we ask the Zivid API to copy data directly from the GPU into this memory location.
+
+```cpp
+const auto pointCloud = frame.pointCloud(); /* Get a handle to the full point cloud on the GPU. */
+
+auto rgba = cv::Mat(pointCloud.height(), pointCloud.width(), CV_8UC4); /* Allocate an OpenCV matrix with
+an appropriate size. */
+
+auto *dataPtr = reinterpret_cast<Zivid::ColorRGBA *>(rgba.data); /* Cast the OpenCV data pointer to
+ColorRGBA* so the Zivid API can understand which data to copy. */
+
+pointCloud.copyData(dataPtr); /* Copy RGBA data straight into the OpenCV memory buffer. */
+```
+
+## Transform
+
+You may want to change the point cloud's origin from the camera to the robot base frame or scale the point cloud to e.g. change it from millimeters to meters. This can be done by transforming the point cloud using ```Zivid::PointCloud::transform(Zivid::Matrix4x4)``` ([go to source][transform]).
+
+```cpp
+pointCloud.transform(transformationMatrix);
+```
+
+## Conclusion
+
+This tutorial shows how to use the Zivid SDK to extract the point cloud, manipulate it, transform it, and visualize it.
+
+[//]: ### "Recommended further reading"
+
+[installation-instructions-url]: ../../README.md#instructions
+[frame-from-file]:Basic/FileFormats/ReadIterateZDF/ReadIterateZDF.cpp#L15-L17
+[frame-capture]:../Camera/Basic/Capture/Capture.cpp#L28
+[capture-tutorial]:../Camera/Basic/CaptureTutorial.md#L158
+[point-cloud]:Advanced/Downsample/Downsample.cpp#L181
+[transform]:Applications/Advanced/MultiCamera/StitchByTransformationFromZDF/StitchByTransformationFromZDF.cpp
```

Comment:
This link doesn't work, because it gets "Applications" twice in the path. I think this is because the paths are relative to the location of this file (notice how you don't write "Applications/" in the other links here.

Reply:
Ah you fixed this in the meantime :)

Reply:
yes, done :)